exercises/pascals-triangle/PascalsTriangleTest.fs

```diff
@@ -22,9 +28,6 @@ let ``Four rows`` () =
     triangle 4 |> should equal [[1]; [1; 1]; [1; 2; 1]; [1; 3; 3; 1]]
 
 [<Fact(Skip = "Remove to run test")>]
-let ``Five rows`` () =
-    triangle 5 |> should equal [[1]; [1; 1]; [1; 2; 1]; [1; 3; 3; 1]; [1; 4; 6; 4; 1]]
+let ``Negative rows`` () =
+    (fun () -> triangle -1 |> ignore) |> should throw typeof<System.ArgumentOutOfRangeException>
```

Comment:
I would like to encode an error as `None`.

Reply:
Changed it to use `int list list option` to return the result and `None` for error cases. 
- Made changes from `should be Empty` to `should equal (Some ([]: int list list))` for empty test case. 
https://stackoverflow.com/questions/23989847/fsunit-should-equal-fails-on-some 
- Overriding ToTestMethodBodyAssertTemplate to use only Assert Template.